ros2topic/test/test_use_sim_time.py

```diff
@@ -97,7 +97,9 @@ def test_pub_times(self, launch_service, proc_info, proc_output):
         # test method:
         #   speed up sim time to half second, then expect the process
         #   will end at half time compare to system time (5s).
-        self.clock_sec = 0
```

Comment:
this means 1st sim clock data will be published 0, that is equal to the initialized time when ros time is enabled. this means, we already miss one clock update using simulation time for the test. this test expects 5 clock events during 3 seconds with 2Hz, that means we have `0.5`, `1.0`, `1.5`, `2.0` and `2.5` (spin should return before `3.0` is published). having the starting time from `1.0` makes sure the 1st clock data will wake up the timer event.